packages/tools/tests/test/audioV2/utils/audioEngineV2.utils.ts

```diff
@@ -127,6 +139,10 @@ function GetVolumeCurves(result: AudioTestResult): Float32Array[] {
         updateCurve();
 
         result.volumeCurves[channel] = curve;
+
+        // Save the audio test result to the test info so it can be retrieved in `test.afterEach` and attached to the
+        // report if needed.
+        (<any>test.info()).audioTestResult = result;
```

Comment:
as tests don't run in parallel, why not store it to a module-level variable and clear it on beforeEach?
I don't see it in the test report (unless I missed something?)

Reply:
> as tests don't run in parallel, why not store it to a module-level variable and clear it on beforeEach?

I run the tests in parallel locally. Is there a better way to handle this when they're run in parallel?

> I don't see it in the test report (unless I missed something?)

They're only attached to the test report if the test fails. Are you looking at a failed test and not seeing it? I see it on failed test here:
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/16470/merge/testResults/audioV2/index.html#?testId=f7be33e33db07c43512d-006210e4ac8d8df7aca1

Reply:
If you run them in parallel then forget it :-) it's all good.